packages/theme/package.json

```diff
@@ -35,8 +35,8 @@
   "dependencies": {
     "@oclif/core": "3.19.6",
     "@shopify/cli-kit": "3.57.0",
-    "@shopify/theme-check-node": "2.2.2",
-    "@shopify/theme-language-server-node": "1.8.3",
+    "@shopify/theme-check-node": "2.3.0",
+    "@shopify/theme-language-server-node": "1.9.0",
```

Comment:
Ah not yet since I'm going to change things. 1.10 is a breaking change we don't need in the wild. 🙃 I designed the API poorly 🙈 Thought I could bypass the "factory business", but TAE and monorepos reminded me that I can't.